packages/usa-banner/src/usa-banner.twig

```diff
@@ -12,32 +12,49 @@
   <div class="usa-accordion">
     <header class="usa-banner__header">
       <div class="usa-banner__inner">
-        <div class="grid-col-auto">
-          <img class="usa-banner__header-flag" src="./img/us_flag_small.png" alt="U.S. flag">
-        </div>
-        <div class="grid-col-fill tablet:grid-col-auto">
+        <img class="usa-banner__header-flag" src="./img/us_flag_small.png" alt="U.S. flag">
+        <div class="usa-banner__header-content">
```

Comment:
Had to add in a container to group all text content and expand button.  

Equivalent to the previous div on line 18 `<div class="grid-col-fill tablet:grid-col-auto">`